test/integration/revisit_page_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+GovukTest.configure
+
+class RevisitPageTest < ActionDispatch::SystemTestCase
```

Comment:
Sorry I think I messed up the rebase or it weirdly auto rebased in there. There was a deeper problem which is that this test has been removed on `master` [here](https://github.com/alphagov/smart-answers/commit/bcc17a5c9125cd3009c9a5e6c67b769df6802210#diff-cce7934a0c0d19aaec3e6a8ffb44b0f8c6a6e7ef36005a3b2bd36612fdddcbd7) as these minitest integration tests have been replaced with RSpec feature tests. I updated the new feature test `User changes their answer to previous question` to use `Back` rather than change as part of this PR. 

The prefilling integration tests have been lost and need to be re-written in RSpec... but given we are going to split this out into separate PR we can leave for now. I saved the old integration test code locally... will also paste here

```
test "back link returns previously visited page with prefilled checkbox answer" do
    visit "find-coronavirus-support/s"
    within "legend" do
      assert_page_has_content "What do you need help with because of coronavirus?"
    end

    check("Getting food or medicine", visible: false)
    click_on "Continue"
    assert_page_has_content "Getting food"
    click_on "Back"
    assert_page_has_content "What do you need help with because of coronavirus?"

    checked_box = find(:css, "[value='getting_food']", visible: false)
    unchecked_box = find(:css, "[value='feeling_unsafe']", visible: false)

    assert_equal(checked_box.checked?, true)
    assert_equal(unchecked_box.checked?, false)
  end

  test "back link returns previously visited page with prefilled radio button answer" do
    visit "find-coronavirus-support/s"
    within "legend" do
      assert_page_has_content "What do you need help with because of coronavirus?"
    end

    check("Getting food or medicine", visible: false)
    click_on "Continue"
    assert_page_has_content "Getting food"
    choose("Yes", visible: false)
    click_on "Continue"
    assert_page_has_content "Getting food or medicine"
    click_on "Back"
    assert_page_has_content "Getting food"

    checked_button = find(:css, "[value='yes']", visible: false)
    unchecked_button = find(:css, "[value='no']", visible: false)

    assert_equal(checked_button.checked?, true)
    assert_equal(unchecked_button.checked?, false)
  end

  def assert_page_has_content(text)
    assert page.has_content?(text), "'#{text}' not found in page"
  end
  ```